### PR TITLE
Added category container, footer component

### PR DESCRIPTION
**Pull request by tacmoktan**
## Building Job Category component in front page
1. created `src/components/CategoryComponent` (**displays Job Categories**)
2. Made image responsive in `pages/global/home/index.js`

View and suggest changes if required.

### CODE REVIEW

File: src/pages/global/Home/index.js [hunk 1]
```diff
@@ -1,9 +1,30 @@
 import React from 'react'
 import workImage from 'assets/images/work.jpg'
+import { makeStyles } from '@material-ui/core/styles';
+//components
+import { CategoryContainer } from "../../../components";
```
Comment (subeshb1) [inline src/pages/global/Home/index.js]: can be used as
```
import { CategoryContainer } from "components";
```

I've added a setting to make src the root path.

Comment (tacmoktan) [reply src/pages/global/Home/index.js]: Ok, will make changes.

**Comment by tacmoktan** [comment]
1. **Title, Font awesome icons, Googlefonts** added in `public/index.html`
2. **Footer** components added in `src/components/Footer/index.js`
3. **NavBar** styling changed in `src/components/NabBar/index.js`

**Comment by tacmoktan** [comment]
## Buiding Sign Up form
1.  Created **Sign Up** component in `src/pages/auth/signup/index.js`
2. **Route** for signup form added in `src/App.js`.
3. Changed Login form **styles** in `src/pages/auth/login/index.js `.

**Pull request merged**